custom/icds_reports/migrations/0158_update_agg_awc_monthly_view_visit_fields.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 1.11.16 on 2019-10-10
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0157_updated_unique_constraint'),
```

Comment:
I think you would need to update the migration file.

Reply:
done